app/assets/javascripts/discourse/controllers/edit-topic-status-update.js.es6

```diff
@@ -3,20 +3,46 @@ import ModalFunctionality from 'discourse/mixins/modal-functionality';
 import TopicStatusUpdate from 'discourse/models/topic-status-update';
 import { popupAjaxError } from 'discourse/lib/ajax-error';
 
-const CLOSE_STATUS_TYPE = 'close';
-const OPEN_STATUS_TYPE = 'open';
-
 export default Ember.Controller.extend(ModalFunctionality, {
+  closeStatusType: 'close',
+  openStatusType: 'open',
+  publishToCategoryStatusType: 'publish_to_category',
   updateTimeValid: null,
   updateTimeInvalid: Em.computed.not('updateTimeValid'),
   loading: false,
   updateTime: null,
   topicStatusUpdate: Ember.computed.alias("model.topic_status_update"),
   selection: Ember.computed.alias('model.topic_status_update.status_type'),
-  autoReopen: Ember.computed.equal('selection', OPEN_STATUS_TYPE),
-  autoClose: Ember.computed.equal('selection', CLOSE_STATUS_TYPE),
-  disableAutoReopen: Ember.computed.and('autoClose', 'updateTime'),
-  disableAutoClose: Ember.computed.and('autoReopen', 'updateTime'),
+
+  @computed('selection', 'openStatusType')
```

Comment:
This is a bit odd to me -- `openStatusType` used to be a local `const` but now it's a computed property, but aren't we 100% sure the value will never change? 

I think it makes a lot more sense for the CP to be based on `selection` alone and just `=== 
 OPEN_STATUS_TYPE`